nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/calculator/PointCalculator.java

```diff
@@ -24,13 +26,17 @@
 import java.util.Map.Entry;
 import java.util.Objects;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class PointCalculator {
 
     public static final String ERROR_MSG_EXTRACT_PUBLICATION_CONTEXT = "Could not extract publication channel for "
                                                                        + "instanceType {}, candidate: {}";
+    public static final String COUNTRY_CODE_NORWAY = "NO";
```

Comment:
I have seen this in two PRs, possibly some kind of constant…possibly even from a standard library if ISO 3166-2 :)

Reply:
I haven't found a standard library with Norways country code (with standard ISO 3166-2). java.util.Locale does not include Norway i think. Do you know of one? Should i put it in NVA commons?